### ISSUE TITLE
Add marker comments when generating code

### ISSUE THREAD

**Issue by seanf** [RobinBlomberg/kysely-codegen#114]
It would be good if kysely-codegen could add a marker comment at the top of the file when generating code. For instance:

```
// This file was generated automatically by kysely-codegen.
// Please do not edit it by hand.
```

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/RobinBlomberg) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/RobinBlomberg/kysely-codegen/issues/114">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/RobinBlomberg/kysely-codegen/issues/114/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/RobinBlomberg/kysely-codegen/issues/114/pledge.svg">
</picture>
</a>

**Comment by clemmy** [comment]
This would be nice as a part of the library.

My hack for this currently is:

```
kysely-codegen --out-file src/db/types.d.ts && echo -e \"// This is an auto-generated file. Do not edit it directly.\n// See 'Generating DB schema types for Kysely' in README.\n\n$(cat src/db/types.d.ts)\" > src/db/types.d.ts
```

**Comment by RobinBlomberg** [comment]
Should be released in [kysely-codegen@0.16.0](https://github.com/RobinBlomberg/kysely-codegen/releases/tag/0.16.0)! 🚀
